adobe_vipm/flows/sync.py

```diff
@@ -160,7 +176,7 @@ def sync_agreement_prices(
         adobe_client,
         customer,
         agreement,
-        subscriptions_for_update=tuple(s[1]["subscriptionId"] for s in subscriptions_for_update),
+        subscriptions_for_update={s[1]["subscriptionId"] for s in subscriptions_for_update},
```

Comment:
I'm getting lost on `_get_subscriptions_for_update` cause it doesn't get subscriptions, it also terminates and updates it. So the naming/function/flow is really messy here :-( And also, I didn't get how do you solve my previous comment about deployments and that customer subscriptions contains subscription for all the deployments, not just for particular main agreement or deployment agreement

Reply:
For now deployments are not being included. This is ticket no https://softwareone.atlassian.net/browse/MPT-11081